apps/tangle-dapp/types/bridge.ts

```diff
@@ -56,19 +63,23 @@ export type BridgeTokenType = {
 };
 
 export enum BridgeWalletError {
-  MismatchEvm = 'WALLET_MISMATCH_EVM',
-  MismatchSubstrate = 'WALLET_MISMATCH_SUBSTRATE',
-  EvmWrongChain = 'WALLET_EVM_WRONG_CHAIN',
+  WalletMismatchEvm = 'WALLET_MISMATCH_EVM',
+  WalletMismatchSubstrate = 'WALLET_MISMATCH_SUBSTRATE',
+  NetworkMismatchEvm = 'NETWORK_MISMATCH_EVM',
+  NetworkMismatchSubstrate = 'NETWORK_MISMATCH_SUBSTRATE',
 }
 
 export enum BridgeTxState {
+  Initializing = 'Initializing', // The tx is being initialized
+
   Sending = 'Sending', // The user is signing the tx
 
-  Indexing = 'Indexing', // The tx is being indexed by Sygma
+  Indexing = 'Indexing', // The tx is being indexed (Sygma txs only)
 
-  Pending = 'Pending', // The tx is done indexing but still pending
+  Pending = 'Pending', // The tx is done indexing but still pending (Sygma txs only)
```

Comment:
To make it more explicit on the code rather than on comments that these options are Sygma-only, maybe rename it to `SygmaOnlyPending` or `SygmaPending`?